src/components/TeacherComponents/SearchFilters/SearchFilters.tsx

```diff
@@ -1,87 +1,118 @@
 import { FC } from "react";
 import { OakP, OakFlex } from "@oaknational/oak-components";
+import styled from "styled-components";
 
 import SearchFilterCheckbox from "@/components/TeacherComponents/SearchFilterCheckbox";
 import { UseSearchFiltersReturnType } from "@/context/Search/search.types";
 import { FilterTypeValueType } from "@/browser-lib/avo/Avo";
 
 type SearchFiltersProps = {
   searchRefined: (filterType: FilterTypeValueType, filterValue: string) => void;
+  onFilterClick: (filter: string | undefined) => void;
+  lastFocussedFilter: string | null;
 } & UseSearchFiltersReturnType;
 
+const StyledFieldset = styled.fieldset`
```

Comment:
Could this be reusable and live in components ?

Reply:
yeah for sure if we need it again we should do that, for now I don't think it's worth the effort though